trogon/widgets/form.py

```diff
@@ -159,49 +163,46 @@ def _form_changed(self) -> None:
         )
 
         root_command_data = parent_command_data
-        try:
-            for command in path_from_root:
-                option_datas = []
-                # For each of the options in the schema for this command,
-                # lets grab the values the user has supplied for them in the form.
-                for option in command.options:
-                    parameter_control = self.query_one(
-                        f"#{option.key}", ParameterControls
-                    )
-                    value = parameter_control.get_values()
-                    for v in value.values:
-                        assert isinstance(v, tuple)
-                        option_data = UserOptionData(option.name, v, option)
-                        option_datas.append(option_data)
-
-                # Now do the same for the arguments
-                argument_datas = []
-                for argument in command.arguments:
-                    form_control_widget = self.query_one(
-                        f"#{argument.key}", ParameterControls
-                    )
-                    value = form_control_widget.get_values()
-                    # This should only ever loop once since arguments can be multi-value but not multiple=True.
-                    for v in value.values:
-                        assert isinstance(v, tuple)
-                        argument_data = UserArgumentData(argument.name, v, argument)
-                        argument_datas.append(argument_data)
-
-                assert all(isinstance(option.value, tuple) for option in option_datas)
-                assert all(
-                    isinstance(argument.value, tuple) for argument in argument_datas
+        for command in path_from_root:
```

Comment:
Diff looks big here, but all I did was remove the redundant `try` which was just re-raising, and decent the code. There are no actual changes.